src/addAllAssetsToCompilation.js

```diff
@@ -31,7 +31,16 @@ function resolveOutput(compilation, addedFilename, outputPath) {
 async function addFileToAssets(
   compilation,
   htmlPluginData,
-  { filepath, typeOfAsset = 'js', includeSourcemap = true, hash = false, publicPath, outputPath, files = [] }
+  {
+    filepath,
+    typeOfAsset = 'js',
+    includeSourcemap = true,
+    includeGzipped = true,
```

Comment:
Ah I see. Hmm I'm not sure about that approach, because that might be too greedy and match too many files. I think I like the array approach, but also if it just ignored missing files?